src/familiar/dropFamiliars.ts

```diff
@@ -171,7 +217,7 @@ export function getAllDrops(fam: Familiar): { expectedValue: number; expectedTur
   const returnValue = [];
 
   for (let i = current; i < expected.length; i++) {
```

Comment:
the `expected.length` here may cause a problem

Reply:
aargh why do functions have a length of 0